package.json

```diff
@@ -67,6 +67,7 @@
     "angular:dev:deploy": "./scripts/dev.sh",
     "angular:website:news": "yarn --cwd packages/angular run website:news",
     "angular:website:build": "yarn --cwd packages/angular run website:build",
+    "angular:website:functions": "netlify-lambda build scripts/netlify",
```

Comment:
I understand that this is the proposed fix for the svg downloads on the website. But I'm not clear on why it is the best fix or why the downloads wound up in a broken state.

In the future, it would be helpful to put that information in the PR.

Reply:
I think that at some point when we started moving scripts for the new website, we removed some of the scripts for the old one. And we no longer add the `handlers` for Netlify how to handle the download of the icons.

But yes, adding information about why things don't work and what part of the issues will be fixed with the PR is a good thing and something that we miss a lot of times.

For me, it was clear because I spend some time debugging on that, but that is not the case for everyone :)